tests/transport/spdp/rtps_disc.ini

```
@@ -0,0 +1,14 @@
+[common]
+DCPSDebugLevel=10
+
+[domain/23]
+DiscoveryConfig=uni_rtps
+
+[rtps_discovery/uni_rtps]
+SedpMulticast=0
+ResendPeriod=2
+MaxSpdpSequenceMsgResetChecks=5
+
+[transport/the_rtps_transport]
+transport_type=rtps_udp
+use_multicast=0
```

```suggestion
use_multicast=0
```
(current file doesn't end in a newline)